deploy/ansible/provisioning.yml

```diff
@@ -1,18 +1,39 @@
 ---
-- hosts: all
+- hosts: localhost
+  connection: local
+
+  tasks:
+    - name: Create Variable File for Each host
+      template: src=template.yml.j2 dest=host_vars/{{ item.1 }}
+      with_indexed_items: "{{ groups['hosts'] }}"
+
+
+- hosts: hosts
   vars:
     user: iroha #your user on the remote machine
     tmp: "{{9999999 | random | to_uuid }}"
 
     postgres_host: iroha_postgres
-    postgres_user: '{{ user }}'
-    postgres_passwd: "{{ tmp[:8] }}"
+    postgres_user: iroha
     postgres_port: 5432
 
     redis_host: iroha_redis
     redis_port: 6379
 
+    IROHA_HOME: '{{playbook_dir}}/../../'
+
+  pre_tasks:
+    - name: Load Host Specific Generated Variables
+      include_vars: host_vars/{{ inventory_hostname }}
+
+
   tasks:
+
+    - name: Set postgres password
+      set_fact:
+        postgres_passwd: '{{ tmp[:8] }}'
```

Comment:
Seems that it always 8 symbols length, or am I wrong?
If so that's quite weird

Reply:
Why? What length should it be?

Reply:
Seems that psql [can handle](https://laurent-leturgez.com/2017/05/04/how-userrole-password-security-is-managed-in-oracle-vs-postgres/) up to 995 symbols in password, though that too much I think.
Imo 64 chars is fine

Reply:
Let it be 32

Reply:
Let it be 32